.circleci/config.yml

```diff
@@ -137,72 +137,71 @@ workflows:
             tags:
               only: /^v.*/
 
-      # - architect/push-to-app-collection:
-      #     name: push-policies-shared-to-aws-app-collection
-      #     context: architect
-      #     app_name: "policies-shared"
-      #     app_namespace: "giantswarm"
-      #     app_collection_repo: "aws-app-collection"
-      #     requires:
-      #       - push-policies-shared-to-catalog
```

Comment:
Why not push to shared as well? Would be nicer later on.

Reply:
`policies-shared` was the name of the old app

Reply:
Ah do you mean https://github.com/giantswarm/shared-app-collection? Is that used? I'm fine to push there but wasn't sure what the delivery guarantees were at this point

Reply:
Delivery guarantees are not great AFAIK but there was agreement to use it.

Reply:
Added shared app collection as well